cmd/applyconfig/applyconfig_test.go

```diff
@@ -270,11 +293,18 @@ func TestAsTemplate(t *testing.T) {
 		},
 	}
 
+	original := os.Getenv("image")
+	if err := os.Setenv("image", "docker.io/redis"); err != nil {
```

Comment:
This is not threadsafe, we can't  be doing that in parallel test

Reply:
I have to admit that I have not considered parallel testing when writing the code.
Thanks, @bbguimaraes